app/views/devise/registrations/new.html.erb

```diff
@@ -0,0 +1,35 @@
+
+<h2>Sign up</h2>
+
+<%= form_for(resource, as: resource_name, url: registration_path(resource_name)) do |f| %>
+  <%= render "devise/shared/error_messages", resource: resource %>
+  <div class="field">
+    <%= f.label :email %>
+    <%= f.email_field :email, autofocus: true, autocomplete: "email" %>
+  </div>
+
+  <div class="field">
+    <%= f.label :name %>
+    <%= f.text_field :name, autofocus: true, autocomplete: "email" %>
+  </div>
+ 
+
+  <div class="field">
+    <%= f.label :password %>
+    <% if @minimum_password_length %>
+    <em>(<%= @minimum_password_length %> characters minimum)</em>
+    <% end %><br />
+    <%= f.password_field :password, autocomplete: "new-password" %>
+  </div>
+
+  <div class="field">
+    <%= f.label :password_confirmation %><br />
+    <%= f.password_field :password_confirmation, autocomplete: "new-password" %>
+  </div>
+
+  <div class="actions">
+    <%= f.submit "Sign up" %>
+  </div>
+<% end %>
+
+<%= render "devise/shared/links" %>
```

Comment:
- [ ] Great work implementing some changes requested by the previous reviewer. Kindly make the **_sign-up_** page and **_log-in_** page match this [wireframe](https://github.com/microverseinc/curriculum-rails/blob/main/blog-app/images/blog_login.png) provided. They used placeholders and not labels for their form inputs and most links are not present in the wireframe. Kindly implement it.

### Your Design

![image](https://user-images.githubusercontent.com/77106962/174902667-c3dadf71-00d3-43f5-a417-c861662b4d8b.png)

### Wiireframe

![image](https://user-images.githubusercontent.com/77106962/174902569-46f9db9e-a9da-4e89-9a1d-dcffad803bc0.png)